src/pages/community-champions.js

```diff
@@ -353,6 +359,90 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    color: ${({ theme }) => theme.colorMap.greyDarkOne};
+    margin-bottom: ${LINE_BOTTOM_MARGIN};
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const GreenBulletedListWithGreyText = styled(GreenBulletedList)`
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+    p {
+        color: ${({ theme }) => theme.colorMap.greyLightTwo};
+    }
```

Comment:
With the changes to the bulleted list component can we pass a new component via the prop?